compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/PackToIntrinsics.cpp

```diff
@@ -97,8 +97,10 @@ void PackToIntrinsicsPass::runOnOperation() {
 
   // Run layout propagation patterns to pull in adjacent un-configured ops.
   RewritePatternSet patterns(context);
-  linalg::ControlPropagationFn control = [](Operation *op) -> bool {
-    return !getLoweringConfig(op);
+  linalg::ControlPropagationFn control = [](OpOperand *opOperand) -> bool {
+    Operation *producer = opOperand->get().getDefiningOp();
+    Operation *consumer = opOperand->getOwner();
+    return !getLoweringConfig(producer) || !getLoweringConfig(consumer);
```

Comment:
@qedawkins is the change correct? Do we check both operations, or do we only check producer or consumer?

Reply:
I think we want `&&` because one of the operations will be a `pack`/`unpack` and the other will be the to-be-packed op.